aria/storage/modeling/instance_elements.py

```diff
@@ -193,7 +193,7 @@ def service_instance(cls):
         return cls.many_to_one_relationship('service_instance')
 
     @declared_attr
-    def interfaces(cls):
+    def interface(cls):
```

Comment:
but this is a service_instance to interface operation. Since you've changed it from plural to singular, I've assumed that it's a one_to_one relationship.